.github/workflows/nightly-normal-session-e2e-tests.yml

```diff
@@ -210,6 +210,26 @@ jobs:
           ALEPH_NODE_BINARY: ../target/release/aleph-node
         run: ./.github/scripts/test_major_sync.sh
 
+  run-force-reorg-test:
+    needs: [build-production-node-and-runtime]
+    name: Run force reorgs test
+    runs-on: [self-hosted, Linux, X64, medium]
```

Comment:
It means it can be only run on a  `self-hosted` runner?

Reply:
default github runner and self-hosted small instance were not powerful enough for this test (8 nodes running at the same time)